cocos/ui/widget-manager.ts

```diff
@@ -70,7 +70,7 @@ function align (node: Node, widget: Widget): void {
     let x = _tempPos.x;
     let y = _tempPos.y;
     const anchor = uiTrans.anchorPoint;
-    const scale = node.getScale();
+    const scale = node.scale;
```

Comment:
It is good now. But it may cause a potential problem that, scale may be changed in future, as it is a reference to Node's member variable.

Reply:
This reference only exists within the align function, and it is not held long-term.

Reply:
Maybe can add some comment.

Reply:
Sure.